src/components/modules/SpatialAirVis/SpatialAirVis.tsx

```diff
@@ -73,17 +82,33 @@ function SpatialAirVis() {
           Submit
         </button>
       </form>
-      {avgData && submitted && (
-        <ul className='flex flex-wrap list-none p-0 m-auto w-1/2 my-6'>
-          {Object.entries(avgData).map(([key, value]) => (
-            <React.Fragment key={key}>
-              <li className='w-1/2 p-4 border text-center'>{key}</li>
-              <li className='w-1/2 p-4 border text-center bg-gray-200'>
-                {value}
-              </li>
-            </React.Fragment>
-          ))}
-        </ul>
+      {submitted && (
+        <table className='w-full border border-collapse'>
+          <thead>
+            <tr>
+              {Object.keys(initialAirQuality).map(key => (
+                <th key={key} className='border border-gray-400 px-4 py-2'>
+                  {key}
+                </th>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {avgData.map((row, index) => (
+              <tr
+                key={Symbol(index).toString()}
+                className={index % 2 === 1 ? 'bg-gray-200' : ''}>
+                {Object.values(row).map((value, colIndex) => (
+                  <td
+                    key={Symbol(colIndex).toString()}
```

Comment:
```suggestion
                {Object.entries(row).map(([columnName, value]) => (
                  <td
                    key={columnName}
```
The problem same as above